guides/common/modules/con_provisioning-virtual-machines-on-kvm.adoc

```diff
@@ -17,10 +17,10 @@ You can use KVM provisioning to create hosts over a network connection or from a
 include::snip_common-compute-resource-prereqs.adoc[]
 * A {SmartProxyServer} managing a network on the KVM server.
 Ensure no other DHCP services run on this network to avoid conflicts with {SmartProxyServer}.
-For more information about network service configuration for {SmartProxyServer}s, see {ProvisioningDocURL}Configuring_Networking_provisioning[Configuring Networking] in _{ProvisioningDocTitle}_.
```

Comment:
Are we talking about this type of change? Replaced the attribute with `Provisioning Guide`, for example.

@Lennonka @ekohl @adamlazik1

Reply:
Yes. This kind of change shouldn't be here.

Reply:
Addressed :+1: